.github/workflows/int-and-connected-test-run.yml

```diff
@@ -73,8 +73,8 @@ jobs:
           # extract service account credentials
           base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
           base64 --decode <<< ${{ secrets.B64_RBS_APPLICATION_CREDENTIALS }} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
-          # assemble code and run connected tests
-          ./gradlew assemble -w testConnected --scan
```

Comment:
I think Phil mentioned his preference is to have `--scan` come after `./gradlew` so it doesn't look like a parameter to `testConnected`, but I don't have a strong opinion about it.

Reply:
Link to previous comment by @pshapiro4broad : https://github.com/DataBiosphere/jade-data-repo/pull/1781#discussion_r1731390291

Reply:
Thanks for following up here! I don't have a strong opinion either, so I'll switch it back.